schemas/2018-05-01-preview/Microsoft.Insights.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-05-01-preview/microsoft.insights.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "microsoft.insights",
+  "description": "microsoft insights Resource Types",
+  "resourceDefinitions": {
+    "ProactiveDetectionConfigs": {
```

Comment:
>ProactiveDetectionConfigs [](start = 5, length = 25)

please add the new resource types to 2015-01-01/deploymentTemplate.json and 2014-04-01-preview/deploymentTemplate.json